kvmagent/kvmagent/plugins/host_model_mount_plugin.py

```
@@ -0,0 +1,646 @@
+                                "Mount point %s already mounted with different storageUrl. "
+                                "Registry has [%s], request has [%s]. Use explicit remount to change." % (
+                                    mount_point, _mask_url(existing_url), _mask_url(cmd.storageUrl)))
+                            # Don't overwrite - just log and continue
+                    else:
+                        # No registry entry exists - check if mounted source matches
+                        actual_source = _get_mount_source(mount_point)
+                        if actual_source and actual_source == cmd.storageUrl:
+                            logger.info("Mount point %s already mounted with matching URL, saving registry entry" % mount_point)
+                            try:
+                                _save_mount_registry_entry(cmd.modelCenterUuid, cmd.storageUrl, mount_point)
+                            except Exception as save_err:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)